docs/using-kubernetes-api-to-create-cf-resources.md

```diff
@@ -80,23 +82,51 @@ subjects:
   - kind: User
     name: my-cf-user
 EOF
-
 ```
-> **Note:** When configuring a `RoleBinding`, it is possible to specify multiple `subjects` for a single binding. However, to maintain compatibility with CF CLI it is necessary to maintain a 1:1 ratio between `RoleBindings` and `Users`/`ServiceAccounts`.
 
-To assign a `SpaceDeveloper` role for a user in a space
+#### Grant a user or service account admin-level access
```

Comment:
Do we have any examples of a ServiceAccount role binding? Or at least what the subject should look like for those?